packages/neon/neon/lib/src/pages/home.dart

```diff
@@ -150,17 +149,17 @@ class _HomePageState extends State<HomePage> {
       },
     );
 
-    final body = OptionBuilder<global_options.NavigationMode>(
-      option: _globalOptions.navigationMode,
-      builder: (final context, final navigationMode) {
+    final body = ValueListenableBuilder<global_options.NavigationMode>(
+      valueListenable: _globalOptions.navigationMode,
+      builder: (final context, final navigationMode, final child) {
         final drawerAlwaysVisible = navigationMode == global_options.NavigationMode.drawerAlwaysVisible;
 
         final body = Scaffold(
           key: _scaffoldKey,
           resizeToAvoidBottomInset: false,
           drawer: !drawerAlwaysVisible ? drawer : null,
           appBar: appBar,
-          body: appView,
+          body: child,
```

Comment:
What happened to the appView variable? I don't see that you renamed it or removed it?

Reply:
the child parameter get's specified below (other comment)